src/test/java/edu/cornell/kfs/module/purap/document/service/impl/PurchaseOrderTransmissionMethodDataRulesServiceImplTest.java

```diff
@@ -1,104 +1,183 @@
-		boolean valid = poTransMethodSataRulesService.isCountryCodeValid("");
+		boolean valid = poTransMethodSataRulesService.isCountryCodeValid(EMPTY);
 		assertFalse(valid);
 	}
 	
+	@Test
 	public void testIsStateCodeValid_Success(){
-		boolean valid = poTransMethodSataRulesService.isStateCodeValid("NY");
+		boolean valid = poTransMethodSataRulesService.isStateCodeValid(STATE);
 		assertTrue(valid);
 	}
 	
+	@Test
 	public void testIsStateCodeValid_Fail(){
-		boolean valid = poTransMethodSataRulesService.isStateCodeValid("");
+		boolean valid = poTransMethodSataRulesService.isStateCodeValid(EMPTY);
 		assertFalse(valid);
 	}
 	
+	@Test
 	public void testIsZipCodeValid_Success(){
-		boolean valid = poTransMethodSataRulesService.isZipCodeValid("14950");
+		boolean valid = poTransMethodSataRulesService.isZipCodeValid(ZIP_GOOD);
 		assertTrue(valid);
 	}
 	
+	@Test
 	public void testIsZipCodeValid_Fail(){
-		boolean valid = poTransMethodSataRulesService.isZipCodeValid("");
+		boolean valid = poTransMethodSataRulesService.isZipCodeValid(EMPTY);
 		assertFalse(valid);
 	}
 	
+	@Test
 	public void testIsAddress1Valid_Success(){
-		boolean valid = poTransMethodSataRulesService.isAddress1Valid("line 1 address");
+		boolean valid = poTransMethodSataRulesService.isAddress1Valid(ADDRESS);
 		assertTrue(valid);
 	}
 	
+	@Test
 	public void testIsAddress1Valid_Fail(){
-		boolean valid = poTransMethodSataRulesService.isAddress1Valid("");
+		boolean valid = poTransMethodSataRulesService.isAddress1Valid(EMPTY);
 		assertFalse(valid);
 	}
 	
+	@Test
 	public void testIsCityValid_Success(){
-		boolean valid = poTransMethodSataRulesService.isCityValid("Ithaca");
+		boolean valid = poTransMethodSataRulesService.isCityValid(CITY);
 		assertTrue(valid);
 	}
 	
+	@Test
 	public void testIsCityValid_Fail(){
-		boolean valid = poTransMethodSataRulesService.isCityValid("");
+		boolean valid = poTransMethodSataRulesService.isCityValid(EMPTY);
 		assertFalse(valid);
 	}
 	
+	@Test
 	public void testIsPostalAddressValid_Success(){
-		boolean valid = poTransMethodSataRulesService.isPostalAddressValid("line 1 address", "Ithaca", "NY", "14850", "US");
+		boolean valid = poTransMethodSataRulesService.isPostalAddressValid(ADDRESS, CITY, STATE, ZIP_GOOD, COUNTRY);
 		assertTrue(valid);
 	}
 	
+	@Test
 	public void testIsPostalAddressValid_Fail(){
-		boolean valid = poTransMethodSataRulesService.isPostalAddressValid("line 1 address", "Ithaca", "NY", "64835634856", "US");
+		boolean valid = poTransMethodSataRulesService.isPostalAddressValid(ADDRESS, CITY, STATE, ZIP_BAD, COUNTRY);
 		assertFalse(valid);
 	}
+	
+	private State makeState() {
+		StateBo bo = new StateBo();
+		bo.setCode(STATE);
+		bo.setName("New York");
+		bo.setCountryCode(COUNTRY);
+		return StateBo.to(bo);
+	}
+	
+	private class testablePhoneNumberServiceImpl extends PhoneNumberServiceImpl{
+		@Override
+		protected String[] parseFormats() {
+			//This is the string that is in our parameter table
```

Comment:
I'm not a huge fan of this comment (and the one like it below) as I'm not sure it adds a lot of value.